### PR TITLE
feat: verify current password and fix signout of all session leading …

### PR DESCRIPTION
**Pull request by danthecm**
…to an error page

# Fixes Issue/Linear Ticket

( This could be an existing issue or a linear ticket )

**My PR Closes #issue_number_here** or
**My PR Fixes ID-linear_ticket_number_here**

# Changes proposed

## What were you told to do?

Fix the save password button leading to an error page in https://zuri.chat/admin/settings/accountsProfile
Sign out of all session button Leads to an error page in https://zuri.chat/admin/settings/accountsProfile

## What did you do?

Create a handler for the save password, verify the current password from api.zuri
create a handler for the sign out of all session button click

# Check List (Check all the applicable boxes)

🚨Please review the [style guide for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/STYLING.md) and [guidelines for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/CONTRIBUTING.md) to this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/ Videos

### CODE REVIEW

File: packages/main/src/components/save-password/SavePassword.jsx [hunk 1]
```diff
@@ -1,27 +1,87 @@
...
-    options: {
-      method: "PATCH",
-      credentials: "include",
+
+  const checkPassword = async password => {
+    const confirmPassword = {
+      password,
+      confirm_password: password
+    };
+    const response = await fetch(`${BASE_API_URL}/auth/confirm-password`, {
```
Comment (billmal071) [inline packages/main/src/components/save-password/SavePassword.jsx:18]: Use axios.

File: packages/main/src/components/save-password/SavePassword.jsx [hunk 2]
```diff
@@ -1,27 +1,87 @@
...
-    endpointUrl: FORM_ENDPOINT
-  });
+      },
+      body: JSON.stringify(confirmPassword)
+    });
+
+    const data = await response.json();
+
+    if (!response.ok) {
+      throw new Error(data.message);
```
Comment (billmal071) [inline packages/main/src/components/save-password/SavePassword.jsx:31]: Use toat notifications to show errors.

**Review by billmal071 — CHANGES_REQUESTED** [review]

File: packages/main/src/components/save-password/SavePassword.jsx (hunk 1 above)
Comment (danthecm) [reply packages/main/src/components/save-password/SavePassword.jsx]: I now using Axios please review again

Comment (danthecm) [reply packages/main/src/components/save-password/SavePassword.jsx]: @billmal071

File: packages/main/src/components/save-password/SavePassword.jsx (hunk 2 above)
Comment (danthecm) [reply packages/main/src/components/save-password/SavePassword.jsx]: I am now using toast please review again @billmal071

**Pull request merged**